opm/parser/eclipse/EclipseState/Grid/TranCalculator.hpp

```diff
@@ -76,6 +76,10 @@ class TranCalculator {
         return this->m_name;
     }
 
+    void clear() {
+        this->actions.clear();
```

Comment:
This does not release the memory of the vector. At least we should suggest this by `this->actions.shrink_to_fit()`

Reply:
I'll add the `shrink_to_fit()` - but mind you this vector does not consume much storage - it is `#tran_modifiers X sizeof( string + enum)`